Added improvements in reverse-proxy-configuration-nginx documentation (#4090)
jenkins-infra/jenkins.io: content/doc/book/system-administration/reverse-proxy-configuration-nginx.adoc

[source,sh]
----
usermod -aG jenkins nginx
----

> If the last command failed due that `nginx` user was not successfully found in the system,
> then you can try adding the `www-data` user to the Jenkins group:

Reviewer: Minor (optional) phrasing change
```suggestion
If the last command failed because the `nginx` user is not defined in the system,
then you can try adding the `www-data` user to the Jenkins group:
```
